"2","Error en gzfile(file, ""wb""): no se puede abrir la conexi√≥n
"
